src/openzeppelin/token/erc20/presets/ERC20.cairo

```diff
@@ -10,140 +10,102 @@ from starkware.cairo.common.uint256 import Uint256
 from openzeppelin.token.erc20.library import ERC20
 
 @constructor
-func constructor{
-        syscall_ptr: felt*,
-        pedersen_ptr: HashBuiltin*,
-        range_check_ptr
-    }(
-        name: felt,
-        symbol: felt,
-        decimals: felt,
-        initial_supply: Uint256,
-        recipient: felt
-    ):
-    ERC20.initializer(name, symbol, decimals)
-    ERC20._mint(recipient, initial_supply)
-    return ()
-end
-
-#
-# Getters
-#
+func constructor{syscall_ptr: felt*, pedersen_ptr: HashBuiltin*, range_check_ptr}(
+    name: felt, symbol: felt, decimals: felt, initial_supply: Uint256, recipient: felt
+) {
+    ERC20.initializer(name, symbol, decimals);
+    ERC20._mint(recipient, initial_supply);
+    return ();
+}
+
+//
+// Getters
+//
 
 @view
-func name{
-        syscall_ptr : felt*,
-        pedersen_ptr : HashBuiltin*,
-        range_check_ptr
-    }() -> (name: felt):
-    let (name) = ERC20.name()
-    return (name)
-end
+func name{syscall_ptr: felt*, pedersen_ptr: HashBuiltin*, range_check_ptr}() -> (name: felt) {
+    let (name) = ERC20.name();
+    return (name,);
```

Comment:
```suggestion
    return ERC20.name();
```
if this works, i'd do the same below
if it doesn't, i'd change `ERC20.name` so it does :)